README.md

```diff
@@ -1,4 +1,4 @@
-# Appeal Planning Decision 
+# Appeal Planning Decision
```

Comment:
It seems you've added a load of spaces to the end of lines in the README files - is this for a reason, or is it done automatically by your IDE? If the latter, be worth installing [EditorConfig](https://editorconfig.org/) as there is a config file for this in the root (.editorconfig)

(I'm genuinely unsure if this is you removing them or adding them - if you're removing them, then that's fine)

Reply:
made a couple of goes at this and.. my editor is absolutely determined that these lines should not have spaces on the end, and seems to have opinions on lines that only contain whitespace..

i think this is technically a very slight improvement? if we end up with crappy commit-wars over whitespace i'll try harder but think this is ok to merge as is

Reply:
A commit war sounds fun.